packages/babel-plugin-emotion/package.json

```diff
@@ -23,6 +23,7 @@
   },
   "devDependencies": {
     "babel-cli": "^6.24.1",
+    "emotion-theming": "^1.0.0",
```

Comment:
This doesn't need to be a devDependency here since it's in jest's `moduleNameMapper`.

Reply:
I’m on vacation until Tuesday, so if you want to get this in sooner feel free to push that change on.

Reply:
No rush. Enjoy your vacation!